### PR TITLE
Add missing field to types/node ReadLineOptions interface

### PR DESCRIPTION
**Pull request by hexpunk**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [ ] `tslint.json` should be present and it shouldn't have any additional or disabling of rules. Just content as `{ "extends": "dtslint/dt.json" }`. If for reason the some rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]`  and not for whole package so that the need for disabling can be reviewed.
- [ ] `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://nodejs.org/docs/latest-v10.x/api/readline.html#readline_readline_createinterface_options
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

If removing a declaration:
- [ ] If a package was never on DefinitelyTyped, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

**Comment by JayAndCatchFire** [comment]
Can I please get help from a maintainer?  The [CI job that failed](https://dev.azure.com/definitelytyped/DefinitelyTyped/_build/results?buildId=31998&view=logs&j=275f1d19-1bd8-5591-b06b-07d489ea915a&t=40b1ee41-44d6-5bba-aa04-4b76a5c732e5&l=97) appears to be an NPM issue, not an issue with my changes.

Here's the failing log for posterity:
```
2020-01-24T23:36:22.8221599Z ##[section]Starting: Run comparisons
2020-01-24T23:36:22.8224571Z ==============================================================================
2020-01-24T23:36:22.8224880Z Task         : Command line
2020-01-24T23:36:22.8225069Z Description  : Run a command line script using Bash on Linux and macOS and cmd.exe on Windows
2020-01-24T23:36:22.8225235Z Version      : 2.163.0
2020-01-24T23:36:22.8225396Z Author       : Microsoft Corporation
2020-01-24T23:36:22.8225527Z Help         : https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/command-line
2020-01-24T23:36:22.8225712Z ==============================================================================
2020-01-24T23:36:22.9460433Z Generating script.
2020-01-24T23:36:22.9481851Z (node:4092) Warning: Use Cipheriv for counter mode of aes-256-ctr
2020-01-24T23:36:22.9482769Z (node:4092) Warning: Use Cipheriv for counter mode of aes-256-ctr
2020-01-24T23:36:22.9483627Z (node:4092) Warning: Use Cipheriv for counter mode of aes-256-ctr
2020-01-24T23:36:22.9484765Z (node:4092) Warning: Use Cipheriv for counter mode of aes-256-ctr
2020-01-24T23:36:22.9485749Z (node:4092) Warning: Use Cipheriv for counter mode of aes-256-ctr
2020-01-24T23:36:22.9486543Z (node:4092) Warning: Use Cipheriv for counter mode of aes-256-ctr
2020-01-24T23:36:22.9487714Z (node:4092) Warning: Use Cipheriv for counter mode of aes-256-ctr
2020-01-24T23:36:22.9487982Z ========================== Starting Command Output ===========================
2020-01-24T23:36:22.9488918Z (node:4092) Warning: Use Cipheriv for counter mode of aes-256-ctr
2020-01-24T23:36:22.9489641Z (node:4092) Warning: Use Cipheriv for counter mode of aes-256-ctr
2020-01-24T23:36:22.9514814Z [command]/bin/bash --noprofile --norc /home/vsts/work/_temp/8d774399-2b26-47fb-a857-23387059be31.sh
2020-01-24T23:36:23.7151039Z Parsing definitions...
2020-01-24T23:36:23.7611834Z Found 6396 packages.
2020-01-24T23:36:23.7612789Z Parsing in parallel...
2020-01-24T23:36:43.8080577Z Parsing took 20.047 s
2020-01-24T23:36:44.3686366Z Installing to /home/vsts/.dts/typescript-installs/2.8...
2020-01-24T23:36:45.3969631Z Installed!
2020-01-24T23:36:45.3970609Z 
2020-01-24T23:36:45.3972315Z Installing to /home/vsts/.dts/typescript-installs/2.9...
2020-01-24T23:36:47.1576445Z Installed!
2020-01-24T23:36:47.1577302Z 
2020-01-24T23:36:47.1581078Z Installing to /home/vsts/.dts/typescript-installs/3.0...
2020-01-24T23:36:48.4061451Z Installed!
2020-01-24T23:36:48.4062177Z 
2020-01-24T23:36:48.4063188Z Installing to /home/vsts/.dts/typescript-installs/3.1...
2020-01-24T23:36:49.6216185Z Installed!
2020-01-24T23:36:49.6218595Z 
2020-01-24T23:36:49.6220874Z Installing to /home/vsts/.dts/typescript-installs/3.2...
2020-01-24T23:36:50.8427095Z Installed!
2020-01-24T23:36:50.8427699Z 
2020-01-24T23:36:50.8434029Z Installing to /home/vsts/.dts/typescript-installs/3.3...
2020-01-24T23:36:52.0607819Z Installed!
2020-01-24T23:36:52.0610562Z 
2020-01-24T23:36:52.0612287Z Installing to /home/vsts/.dts/typescript-installs/3.4...
2020-01-24T23:36:53.5694305Z Installed!
2020-01-24T23:36:53.5694946Z 
2020-01-24T23:36:53.5698634Z Installing to /home/vsts/.dts/typescript-installs/3.5...
2020-01-24T23:36:55.0050940Z Installed!
2020-01-24T23:36:55.0054323Z 
2020-01-24T23:36:55.0055648Z Installing to /home/vsts/.dts/typescript-installs/3.6...
2020-01-24T23:36:56.3053064Z Installed!
2020-01-24T23:36:56.3054550Z 
2020-01-24T23:36:56.3056395Z Installing to /home/vsts/.dts/typescript-installs/3.7...
2020-01-24T23:36:57.4205485Z Installed!
2020-01-24T23:36:57.4206429Z 
2020-01-24T23:36:57.4209026Z Installing to /home/vsts/.dts/typescript-installs/next...
2020-01-24T23:36:58.5327218Z Installed!
2020-01-24T23:36:58.5328433Z 
2020-01-24T23:36:58.8817786Z Comparing node/v13 because it changed...
2020-01-24T23:36:58.8819172Z 
2020-01-24T23:36:58.8819796Z 
2020-01-24T23:36:59.2449298Z No comparable benchmark for node/v13. Checking out master and running one...
2020-01-24T23:36:59.8343878Z Parsing definitions...
2020-01-24T23:36:59.8753151Z Found 6396 packages.
2020-01-24T23:36:59.8754094Z Parsing in parallel...
2020-01-24T23:37:20.9315991Z Parsing took 21.056 s
2020-01-24T23:38:04.1456346Z Error: Error: Command failed: npm install --ignore-scripts --no-shrinkwrap --no-package-lock --no-bin-links
2020-01-24T23:38:04.1457512Z npm ERR! code ENOENT
2020-01-24T23:38:04.1457966Z npm ERR! syscall lchown
2020-01-24T23:38:04.1458626Z npm ERR! path /home/vsts/.npm/_locks/staging-d1b5d13cdc684af1.lock
2020-01-24T23:38:04.1459266Z npm ERR! errno -2
2020-01-24T23:38:04.1459973Z npm ERR! enoent ENOENT: no such file or directory, lchown '/home/vsts/.npm/_locks/staging-d1b5d13cdc684af1.lock'
2020-01-24T23:38:04.1460459Z npm ERR! enoent This is related to npm not being able to find a file.
2020-01-24T23:38:04.1460817Z npm ERR! enoent 
2020-01-24T23:38:04.1461337Z 
2020-01-24T23:38:04.1461958Z npm ERR! A complete log of this run can be found in:
2020-01-24T23:38:04.1462615Z npm ERR!     /home/vsts/.npm/_logs/2020-01-24T23_38_04_012Z-debug.log
2020-01-24T23:38:04.1463026Z 
2020-01-24T23:38:04.1463425Z     at ChildProcess.exithandler (child_process.js:295:12)
2020-01-24T23:38:04.1463840Z     at ChildProcess.emit (events.js:223:5)
2020-01-24T23:38:04.1464412Z     at maybeClose (internal/child_process.js:1021:16)
2020-01-24T23:38:04.1464933Z     at Process.ChildProcess._handle.onexit (internal/child_process.js:283:5)
2020-01-24T23:38:04.1465314Z npm ERR! code ENOENT
2020-01-24T23:38:04.1465675Z npm ERR! syscall lchown
2020-01-24T23:38:04.1466271Z npm ERR! path /home/vsts/.npm/_locks/staging-d1b5d13cdc684af1.lock
2020-01-24T23:38:04.1466895Z npm ERR! errno -2
2020-01-24T23:38:04.1468090Z npm ERR! enoent ENOENT: no such file or directory, lchown '/home/vsts/.npm/_locks/staging-d1b5d13cdc684af1.lock'
2020-01-24T23:38:04.1468591Z npm ERR! enoent This is related to npm not being able to find a file.
2020-01-24T23:38:04.1469002Z npm ERR! enoent 
2020-01-24T23:38:04.1469355Z 
2020-01-24T23:38:04.1469707Z npm ERR! A complete log of this run can be found in:
2020-01-24T23:38:04.1470357Z npm ERR!     /home/vsts/.npm/_logs/2020-01-24T23_38_04_012Z-debug.log
2020-01-24T23:38:04.1471496Z     at Object.execAndThrowErrors (/home/vsts/work/1/s/node_modules/@andrewbranch/definitely-not-slow/node_modules/types-publisher/src/util/util.ts:160:15)
2020-01-24T23:38:04.1471998Z     at processTicksAndRejections (internal/process/task_queues.js:94:5)
2020-01-24T23:38:04.1472685Z     at /home/vsts/work/1/s/node_modules/@andrewbranch/definitely-not-slow/node_modules/types-publisher/src/util/util.ts:58:28
2020-01-24T23:38:04.1473168Z     at async Promise.all (index 0)
2020-01-24T23:38:04.1473843Z     at Object.nAtATime (/home/vsts/work/1/s/node_modules/@andrewbranch/definitely-not-slow/node_modules/types-publisher/src/util/util.ts:53:5)
2020-01-24T23:38:04.1714058Z ##[error]Bash exited with code '1'.
2020-01-24T23:38:04.1757980Z ##[section]Finishing: Run comparisons
```

**Comment by JayAndCatchFire** [comment]
One CI job threw a `ENOENT` error.  The other one had issues with external dependencies for the `assert`/`asserts` definitions.  These are particularly baffling to me as the errors from this run are different from the last run.  (In fact, one of the two CI jobs succeeded on the last run.)  Also, these tests all succeed locally for me with a fresh checkout of my branch.

**Review by Flarna — APPROVED** [review]

**Review by r3nya — APPROVED** [review]

**Comment by uniqueiniquity** [comment]
@JayAndCatchFire sorry about the issue here - there actually is a problem with `assert`, so I'm going to try to resolve that before merging this.

**Pull request merged**
